adafruit_dht.py

```diff
@@ -55,7 +55,7 @@ class DHTBase:
 
     __hiLevel = 51
 
-    def __init__(self, dht11: bool, pin: Pin, trig_wait: int, use_pulseio: bool):
+    def __init__(self, dht11: bool, pin: Pin, trig_wait: int, max_pulses: int, use_pulseio: bool):
```

Comment:
pylint is unhappy with the number of positional arguments here. Here is an alternative that uses a kwarg:

```suggestion
    def __init__(self, dht11: bool, pin: Pin, trig_wait: int, use_pulseio: bool, *, max_pulses: int = 81):
```

Reply:
I've suspected that, but I did not want to change original code style too much. Today I've tried to re-structure code in a more object-objected way. However, that made pylint even less happy. I guess that my OO experience is not really applicable to python after all. Never mind. Accepting your suggestion.